lib/scarpe/widget.rb

```diff
@@ -94,25 +97,26 @@ def to_html
       end
     end
 
+    # This binds a Scarpe callback, handled via a single dispatch point in the document root
     def bind(handler_function_name, &block)
       @@document_root.bind(html_id + "-" + handler_function_name, &block)
     end
 
-    def inner_text=(new_text)
-      @@window.eval("document.getElementById(#{html_id}).innerText = \"#{new_text}\"")
+    def html_inner_text=(new_text)
```

Comment:
Are these for compatibility? Apart from having to refactor the widgets, I think it would be cleaner to just use `webview.inner_text =` instead of proxying all these methods.

Reply:
If you're asking why the "html_" prefix, it would be reasonable to leave it off. I mainly do it because we're going to have other places we use "inner_text" (you can see it in some of the Shoes examples), and I find this makes it clearer that we're updating the HTML DOM. But it'd be reasonable to just use "webview.inner_text =", sure.

Reply:
Ah, sorry, you mean that in, like, para we could use webview.inner_text= instead of calling a Scarpe::Widget method. Yeah, we could do that. I'm now kind of wondering if "webview" is the wrong name for this. But yeah, having the specific widget child class request a DOM insertion point, preferably with a better name than "webview", would be even better. Right now you get one "free" with each widget by calling webview, and I'm not sure that's the right thing, long-term.

Still much better than what we were doing, of course :-)

Reply:
It definitely is! I don't mind this, but I feel like the proxy methods don't add much value :D

Reply:
That's fair. I'll rename "webview" and remove the proxy methods.